contracts/Tokens/VTokens/VToken.sol

```diff
@@ -1699,6 +1866,18 @@ abstract contract VToken is VTokenInterface, Exponential, TokenErrorReporter {
         require(address_ != address(0), "zero address");
     }
 
+    function ensureAccrueInterest() private {
+        require(accrueInterest() == uint(Error.NO_ERROR), "accrue interest failed");
+    }
+
+    function checkAccrueInterest(FailureInfo info) private returns (uint) {
+        uint error = accrueInterest();
+        if (error != uint(Error.NO_ERROR)) {
+            // accrueInterest emits logs on errors, but on top of that we want to log the fact that an attempted reserve factor change failed.
+            return fail(Error(error), info);
```

Comment:
With the proposed implementation this return value is ignored and operations continue as normal even if interest accruals fail. I don't think we want that.

Reply:
[d3bf53c](https://github.com/VenusProtocol/venus-protocol/pull/545/commits/d3bf53c51d56c26305d2ccdfa2d54e57461a1259)